.dependabot/config.yml

```diff
@@ -25,5 +33,9 @@ update_configs:
     commit_message:
       prefix: chore
       include_scope: true
+    automerged_updates:
+      - match:
+          dependency_type: all
+          update_type: 'semver:minor'
```

Comment:
what does this mean here? Our submodule only has commits, no tags

Reply:
https://github.com/libero/scripts/ should be one of the repositories where we never make breaking changes given how entrenched it becomes with builds and deployments, so could probably be always updated then.

Reply:
Never say never.  😛

While we could tag, maybe let's just not auto merge these for now.